### ISSUE TITLE
graphql-flutter not working in flutter web

### ISSUE THREAD

**Issue by minhnhatvdl** [zino-app/graphql-flutter#683]
**Describe the bug**
I am using graphql-flutter v3.0.1 for a project of flutter web. Last week, it worked very well. Now, it no longer works.
<img width="392" alt="Capture d’écran 2020-07-02 à 20 51 24" src="https://user-images.githubusercontent.com/12047824/86398403-d3f69600-bca5-11ea-9586-3280d8bdc960.png">

I am using flutter in channel beta with version 1.12.13+hotfix.6

**Comment by MostafaHassan** [comment]
I got the same error in beta, dev and master. 
Note: I only got the error on web server.
So to reproduce, run flutter build web

**Comment by ductranit** [comment]
I had the same issue. Notice for more details: it looks like there is a problem with dependency connectivity 0.4.9 which uses connectivity_for_web 0.3.0 because I got 404 error for connectivity_for_web.dart

**Comment by GP4cK** [comment]
Here is the root cause: https://github.com/flutter/flutter/issues/60855.
You can also add this to your `pubspec.yml`:

```yml
dependency_overrides:
  connectivity: ^0.4.9
  connectivity_for_web: ^0.3.1
```

**Comment by micimize** [comment]
Closed by #754
